.github/workflows/stock-benchmark.yml

```diff
@@ -0,0 +1,82 @@
+name: Stock benchmark
+on:
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'ts/Stock/**'
+      - 'ts/Data/**'
```

Comment:
No need for data-modules, right?

Reply:
idk, we might wanna remove it, if you think so.

Reply:
I would say keep dependencies at a minimum